packages/core/src/__tests__/ai-brief-consolidated.test.ts

```diff
@@ -30,6 +30,9 @@ describe("getAiBriefContentFromBundle", () => {
           resourceType: "Binary",
           id: "123",
           contentType: "text/plain",
+          meta: {
+            source: "some-other-source",
```

Comment:
This should be the expected AI Brief source, so we can really test it's not returning this resource because of the missing data, right?